modules/fx/arrow-fx/src/main/kotlin/arrow/fx/IO.kt

```diff
@@ -764,8 +699,16 @@ sealed class IO<out E, out A> : IOOf<E, A> {
    * @see [unsafeRunAsyncCancellable] to run in a cancellable manner.
```

Comment:
This link may be broken now

Reply:
Hoping to get docs green in following PR and then we can move on to polishing final API and elaborate in docs.